LearningMissionSimulation/PlaygroundVahe.cs

```diff
@@ -156,19 +196,132 @@ List<Module> GetModuleList()
+                    ClassroomDictionary.Add(classroom.Id, classroom);
+                    ReportSummary("Classrooms", classroomCount);
+                    ReportFooter("Modules generated");
+                }
+            }
+            ReportError(ModuleIdList.ToString(), "classroom");
         }
-        
+
         public void AssignInstructorsToClassrooms()
         {
-            throw new NotImplementedException();
+            if(ActiveInstructorList.Count > 0 && ClassroomDictionary.Count > 0)
+            {
+                foreach (var classroom in ClassroomDictionary.Values)
+                {
+                    Guid moduleId = classroom.Module.Id;
+                    List<Instructor> instructorList;
+                    if (ModuleInstructorListDictionary.ContainsKey(moduleId))
+                    {
+                        ModuleInstructorListDictionary.TryGetValue(moduleId, out instructorList);
+                    }
+                }
+            }
+        }
+
+        void AddInstructorToModuleList(Instructor instructor)
+        {
+            List<Module> moduleList = instructor.ModuleList;
+            foreach (var module in moduleList)
+            {
+                List<Instructor> instructorList;
+                if (ModuleInstructorListDictionary.ContainsKey(module.Id))
+                {
+                    ModuleInstructorListDictionary.TryGetValue(module.Id, out instructorList);
+                    instructorList.Add(instructor);
+                }
+                else
+                {
+                    instructorList = new List<Instructor>() { instructor };
+                    ModuleInstructorListDictionary.Add(module.Id, instructorList);
+                }
+            }
         }
 
         public void RegisterStudentsForClasses()
         {
             throw new NotImplementedException();
         }
 
+        public void SimulationAll(int count)
+        {
+
+            CreateAccounts(count);
+            ActivateAccounts();
+            CreateSubjects(count);
+            CreateModules(count);
+            AssignModulesToStudents();
+            AssignModulesToInstructors();
+            CreateClassrooms(count);
+            AssignInstructorsToClassrooms();
+        }
+
+        #region ReportMethods
+        void ReportHeader(string actionName)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine($" <<<<< {actionName} is started >>>>> \n");
+            Console.ForegroundColor = ConsoleColor.Black;
+        }
+
+        void ReportFooter(string actionName)
+        {
+            Console.ForegroundColor = ConsoleColor.DarkRed;
+            Console.WriteLine($" <<<<< {actionName} is finished >>>>> \n");
+            Console.ForegroundColor = ConsoleColor.Black;
+        }
+
+        void ReportItem(string itemName, string actionName, int itemIndex)
+        {
+            Console.WriteLine($"{actionName} {itemIndex}\n{itemName}");
```

Comment:
{itemIndex}\n{itemName} -?
Is it what you want to do: write itemName on the next line?